D102 not silent for `overrides.overrides` or `overrides.override`

<!--
Thank you for taking the time to report an issue! We're glad to have you involved with Ruff.

If you're filing a bug report, please consider including the following information:

* List of keywords you searched for before creating this issue. Write them down here so that others can find this issue more easily and help provide feedback.
  e.g. "RUF001", "unused variable", "Jupyter notebook"
* A minimal code snippet that reproduces the bug.
* The command you invoked (e.g., `ruff /path/to/file.py --fix`), ideally including the `--isolated` flag.
* The current Ruff settings (any relevant sections from your `pyproject.toml`).
* The current Ruff version (`ruff --version`).
-->

On ruff version 0.6.1, this file:

```python
import typing
import overrides

class Base:
    "Sample class"

    def do_thing(self) -> None:
        "Do a thing."
        pass


class First(Base):
    "Success with typing.override"

    @typing.override
    def do_thing(self) -> None:
        pass


class FailueA(Base):
    "Failure with overrides.override"

    @overrides.override
    def do_thing(self) -> None:
        pass


class FailureB(Base):
    "Success with overrides.overrides"

    @overrides.overrides
    def do_thing(self) -> None:
        pass
```

gets flagged twice with `D102`:

```
$ ruff check sample.py --extend-select D102
sample.py:24:9: D102 Missing docstring in public method
   |
23 |     @overrides.override
24 |     def do_thing(self) -> None:
   |         ^^^^^^^^ D102
25 |         pass
   |

sample.py:32:9: D102 Missing docstring in public method
   |
31 |     @overrides.overrides
32 |     def do_thing(self) -> None:
   |         ^^^^^^^^ D102
33 |         pass
   |

Found 2 errors.
```

and I don't see a way [in the docs](https://docs.astral.sh/ruff/rules/undocumented-public-method/) to extend D102 to treat these decorators from `overrides` like `typing.override`

--- a/crates/ruff_linter/src/rules/pydocstyle/rules/not_missing.rs
+++ b/crates/ruff_linter/src/rules/pydocstyle/rules/not_missing.rs
@@ -218,6 +218,9 @@ impl Violation for UndocumentedPublicClass {
 ///             raise ValueError("Tried to greet an unhappy cat.")
 /// ```
 ///
+/// ## Options
+/// - `lint.pydocstyle.ignore-decorators`
+///
 /// ## References
 /// - [PEP 257 – Docstring Conventions](https://peps.python.org/pep-0257/)
 /// - [PEP 287 – reStructuredText Docstring Format](https://peps.python.org/pep-0287/)
@@ -304,6 +307,9 @@ impl Violation for UndocumentedPublicMethod {
 ///     except ZeroDivisionError as exc:
 ///         raise FasterThanLightError from exc
 /// ```
+///
+/// ## Options
+/// - `lint.pydocstyle.ignore-decorators`
 ///
 /// ## References
 /// - [PEP 257 – Docstring Conventions](https://peps.python.org/pep-0257/)
@@ -401,6 +407,9 @@ impl Violation for UndocumentedPublicPackage {
 /// cat = Cat("Dusty")
 /// print(cat)  # "Cat: Dusty"
 /// ```
+///
+/// ## Options
+/// - `lint.pydocstyle.ignore-decorators`
 ///
 /// ## References
 /// - [PEP 257 – Docstring Conventions](https://peps.python.org/pep-0257/)
@@ -501,6 +510,9 @@ impl Violation for UndocumentedPublicNestedClass {
 ///         self.name: str = name
 ///         self.population: int = population
 /// ```
+///
+/// ## Options
+/// - `lint.pydocstyle.ignore-decorators`
 ///
 /// ## References
 /// - [PEP 257 – Docstring Conventions](https://peps.python.org/pep-0257/)

